src/node/test/merkle_bench.cpp

```diff
@@ -181,6 +183,42 @@ static void serialise_deserialise(picobench::state& s)
   s.stop_timer();
 }
 
+static void serialised_size(picobench::state& s)
+{
+  ccf::MerkleTreeHistory t;
+  vector<crypto::Sha256Hash> hashes;
+  std::random_device r;
+
+  for (size_t i = 0; i < s.iterations(); ++i)
+  {
+    crypto::Sha256Hash h;
+    for (size_t j = 0; j < crypto::Sha256Hash::SIZE; j++)
+      h.h[j] = r();
+
+    hashes.emplace_back(h);
```

Comment:
can we not just append to the tree here instead of creating the hashes vector?

Reply:
Sure, it's just copy pasted from earlier tests.